server.js

```diff
@@ -1,25 +1,24 @@
-const liveServer = require('live-server')
-const isSSR = !!process.env.SSR
-const middleware = []
+const liveServer = require('live-server');
+const isSSR = !!process.env.SSR;
+const middleware = [];
 
 if (isSSR) {
-  const Renderer = require('./packages/docsify-server-renderer/build.js')
+  const { initJSDOM } = require('./test/_helper');
+
+  const dom = initJSDOM();
+  dom.reconfigure({ url: 'https://127.0.0.1:3000' });
+
+  require = require('esm')(module /* , options */);
+
+  const {
+    Renderer,
+    getDefaultTemplate,
+  } = require('./packages/docsify-server-renderer/index');
+
+  debugger;
+
   const renderer = new Renderer({
-    template: `
-  <!DOCTYPE html>
-  <html lang="en">
-  <head>
-    <meta charset="UTF-8">
-    <title>docsify</title>
-    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-    <link rel="stylesheet" href="/themes/vue.css" title="vue">
-  </head>
-  <body>
-    <!--inject-app-->
-    <!--inject-config-->
-  <script src="/lib/docsify.js"></script>
-  </body>
-  </html>`,
```

Comment:
moved to a separate HTML file so it can be r-used (f.e. here, and in the test code).